### ISSUE TITLE
Back button is not 44px minimum for accessibility

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#14605]
## What happened?
Using Google accessibility scanner to determine that the "Back" button does not meet minimum requirements that it be at least 44px X 44px for a touch point.

NOTE:  "Done" button uses the same button, just different text.  See also "VA Find a location" and "Coronavirus FAQ" web views.

## Specs:
- Tested with Pixel 3 simulator and Google Accessibility Scanner.

## Acceptance Criteria
All touch points to be a minimum 44px X 44px.

![backButton.png](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/89f0e17e-f265-4911-8218-860683a6eb71)

**Comment by digitalken** [comment]
Logged against:
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/13864

**Comment by digitalken** [comment]
PASS 10/26/2020 - testing with Android Accessibility scanner and the "back" and "done" buttons are no longer flagged for being too small.
